CHANGELOG.md

```diff
@@ -5,9 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
 - x/yarpctest: Add a retry option to HTTP/TChannel/GRPCRequest.
 - Added `peer/tworandomchoices`, an implementation of the Two Random Choices
   load balancer algorithm.
+- HTTP inbounds can be configured with a shutdown timeout to wait for requests
```

Comment:
Should this be under "Changed", since HTTP inbounds previously just stopped the listener, now we do a proper shutdown. We have a default timeout (and added a configurable timeout) but there's a changed behaviour

Reply:
If you phrase it like "Adds graceful shutdown with configurable timeout per HTTP inbound.", it's a *new* feature, but I can also see it as a simple change to the stop behavior of an inbound. I think the default timeout of 5s will be noticeable to users.

Reply:
yup good points, rephrased it under `Changed`